sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -11,10 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import inspect
 import json
+import tarfile
+import yaml
+import zipfile
+from typing import Callable, Set, List, Text, Dict, Tuple, Any, Union, Optional
```

Comment:
Agreed, it's a long line of an import statement, but this is the recommended Python practice (https://www.python.org/dev/peps/pep-0008/#imports) -- to make imports as narrow and explicit as possible. Wild-card imports like `from typing import *` are strongly discouraged.
And since this was copied down from the KFP compiler I was trying to keep changes to a minimum to make it easier to trace actual code changes in the future.